components/local_date_time/local_date_time.jsx

```diff
@@ -39,13 +40,18 @@ export default class LocalDateTime extends React.PureComponent {
 
         const date = eventTime ? new Date(eventTime) : new Date();
 
+        const title = moment(date);
+        if (enableTimezone) {
+            title.tz(timeZone);
+        }
+
         const timezoneProps = enableTimezone && timeZone ? {timeZone} : {};
 
         return (
             <time
                 className='post__time'
                 dateTime={date.toISOString()}
-                title={date}
+                title={enableTimezone ? title.toString() + ' (' + title.tz() + ')' : title.toString()}
```

Comment:
This formatting is meant to mimic the title formatting today, adding the "friendly" timezone in parentheses after the offset.

Reply:
(a) Not sure what is expected but currently the timezone in parenthesis on me is "Philippine Standard Time" compared to this change of "Asia/Manila".

(b) If props.enableTimezone is true and props.userTimezone is not set (or undefined), text on parenthesis is showing "(undefined)".  Maybe need another condition check,
```suggestion
                title={enableTimezone && title.tz() ? title.toString() + ' (' + title.tz() + ')' : title.toString()}
```
![screen shot 2019-03-06 at 7 15 45 pm](https://user-images.githubusercontent.com/5334504/53877907-613fc100-4045-11e9-998c-6e2ed7a5cf71.png)

Reply:
I've also ripped out the snapshots for this particular component -- they really weren't adding any value here, and I've gone ahead and tested the parts that matter explicitly (namely the `title`).